Cilium-dbg CLI segfault ip get -l reserved:host

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Version

equal or higher than v1.18.7 and lower than v1.19.0

### What happened?

On a k3s single node cluster i get this segfault when listing ips with a filter

```shell
bash-5.3# kubecilium cilium-dbg ip get -l reserved:host
===========================
Labels:
    reserved:host
===========================
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x2ba6fb4]

goroutine 1 [running]:
github.com/cilium/cilium/cilium-dbg/cmd.printIPcacheEntriesBrief({0xc00052c080, 0x3, 0xc000b8fa50?})
        /go/src/github.com/cilium/cilium/cilium-dbg/cmd/ip_get.go:102 +0x214
github.com/cilium/cilium/cilium-dbg/cmd.displayByLabels({0xc0006efbb0, 0x1, 0x1})
        /go/src/github.com/cilium/cilium/cilium-dbg/cmd/ip_get.go:87 +0x2d4
github.com/cilium/cilium/cilium-dbg/cmd.init.func104(0xc0002e8600?, {0xc0004a77e0, 0x0, 0x370c159?})
        /go/src/github.com/cilium/cilium/cilium-dbg/cmd/ip_get.go:41 +0x608
github.com/spf13/cobra.(*Command).execute(0x5e2a120, {0xc0004a77a0, 0x2, 0x2})
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:1019 +0xa91
github.com/spf13/cobra.(*Command).ExecuteC(0x5e2a960)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:1148 +0x46f
github.com/spf13/cobra.(*Command).Execute(...)
        /go/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:1071
github.com/cilium/cilium/cilium-dbg/cmd.Execute()
        /go/src/github.com/cilium/cilium/cilium-dbg/cmd/root.go:45 +0x1a
main.main()
        /go/src/github.com/cilium/cilium/cilium-dbg/main.go:9 +0xf
command terminated with exit code 2
```

### How can we reproduce the issue?

Even with default cilium installation i get this error

### Cilium Version

```shell
bash-5.3# kubecilium cilium-dbg version
Client: 1.18.6 95896696 2026-01-12T16:56:04+00:00 go version go1.24.11 linux/amd64
Daemon: 1.18.6 95896696 2026-01-12T16:56:04+00:00 go version go1.24.11 linux/amd64
```

### Kernel Version

 6.18.10-200.fc43.x86_64

### Kubernetes Version

 v1.35.0+k3s3

### Regression

_No response_

### Sysdump

_No response_

### Relevant log output

```shell

```

### Anything else?

_No response_

### Cilium Users Document

- [ ] Are you a user of Cilium? Please add yourself to the [Users doc](https://github.com/cilium/cilium/blob/main/USERS.md)

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/cilium-dbg/cmd/ip_get.go
+++ b/cilium-dbg/cmd/ip_get.go
@@ -25,6 +25,8 @@ var (
 	ipQueryLabels []string
 )
 
+const InfoNA = "N/A"
+
 var ipGetCmd = &cobra.Command{
 	Use:   "get ( <cidr> |-l <identity labels> )",
 	Short: "Display IP Cache information",
@@ -99,7 +101,15 @@ func printIPcacheEntriesBrief(entries []*models.IPListEntry) {
 	w := tabwriter.NewWriter(os.Stdout, 5, 0, 3, ' ', 0)
 	fmt.Fprintf(w, "IP\tHOST\tIDENTITY\tPOD\tNAMESPACE\n")
 	for _, entry := range entries {
-		fmt.Fprintf(w, "%s\t%s\t%d\t%s\t%s\n", *entry.Cidr, entry.HostIP, *entry.Identity, entry.Metadata.Name, entry.Metadata.Namespace)
+		cidr := InfoNA
+		if entry.Cidr != nil {
+			cidr = *entry.Cidr
+		}
+		identity := InfoNA
+		if entry.Identity != nil {
+			identity = fmt.Sprintf("%d", *entry.Identity)
+		}
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", cidr, entry.HostIP, identity, entry.Metadata.Name, entry.Metadata.Namespace)
 	}
 	w.Flush()
 }

